webapp/src/ArchivalPage.js

```diff
@@ -161,8 +308,20 @@ function ArchivalPage() {
         },
         {
             title: 'Website',
-            'dataIndex': 'Website',
+            dataIndex: 'Website',
             key: 'website',
+        },
+        {
+            title: 'Download Status',
+            key: 'DlStatus',
+            dataIndex: 'DlStatus',
+        },
+        {
+            title: 'Progress',
+            key: 'progress',
```

Comment:
Convention for key naming? Are we starting with uppercase names like with Website and DlStatus?

Reply:
It turns out there's no need for a key field if we have a dataindex. I switched it to uppercase.